specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-02-02-preview/managedClusters.json

```diff
@@ -4320,6 +4324,28 @@
       ],
       "description": "A Kubernetes add-on profile for a managed cluster."
     },
+    "ManagedClusterKMSProfile": {
```

Comment:
IMO we should avoid abbreviating things in the REST API where the abbreviation is not well known. Stuff like VM and OS are ok, but here we should consider `ManagedClusterKeyManagementProfile`

Reply:
fixed